ansto_simplon_api/schemas/configuration.py

```diff
@@ -17,3 +19,62 @@ class SimplonRequestAny(BaseModel):
 
 class SimplonRequestStr(BaseModel):
     value: str
+
+
+class SimplonRequestBool(BaseModel):
+    value: bool
+
+
+class ZMQStartMessage(BaseModel):
+    """
+    Default values matches pre-operations
+    test dataset. To be updated when we collect
+    new datasets to be used as standards.
+    """
+
+    type: str = "start"
+    arm_date: datetime = datetime.now()
+    beam_center_x: float = 2099.46240234375
+    beam_center_y: float = 2119.423828125
+    channels: list[str] = ["1"]
+    count_time: float = 0.010999999940395355
+    countrate_correction_enabled: bool = True
+    countrate_correction_lookup_table: list | None
+    detector_description: str = "Dectris EIGER2 Si 16M"
+    detector_serial_number: str = "E-32-0130"
+    detector_translation: list[float, float, float]
```

Comment:
the annotation should be list[float]. If the type were  a tuple you could specify tuple[float, float, float], otherwise I'm getting an error

Reply:
So, list[float] even though it's always a list of three floats?

Reply:
yeah. If you know exactly how many parameters you get it's better tu use a tuple, then you can set `tuple[float, float, float]`